app/src/main/java/com/framgia/wsm/screen/main/MainViewModel.java

```diff
@@ -283,5 +293,8 @@ private String titleWorkingCalendar() {
         int OFF = 6;
         int COME_LATE_LEAVE_EARLY = 7;
         int WORKSPACE_DATA = 8;
+        int MANAGE_OVERTIME = 9;
+        int MANAGE_OFF = 10;
+        int MANAGE_COME_LATE_LEAVE_EARLY = 11;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.